openfisca_us/parameters/usda/disabled_programs.yaml

```
@@ -0,0 +1,14 @@
+description: Program participation that qualify individuals as disabled under USDA rules
+
+values:
+  2021-01-01:
+  - is_ssi_disabled
+  - ssdi
+  - permanently_disabled_veteran
```

Could you add `is_` to each of these three veteran variables? (Here and in the respective definitions.)
```suggestion
  - is_permanently_disabled_veteran
```